src/Compilers/CSharp/Portable/Errors/ErrorCode.cs

```diff
@@ -1468,6 +1468,8 @@ internal enum ErrorCode
         ERR_VoidInTuple = 8210,
         #endregion more stragglers for C# 7
 
+        #region diagnostics introduced for C# 7.1
```

Comment:
Although I did some compaction on bottom of the list, I just noticed the gap from 8210 to 8300.
Opened [https://github.com/dotnet/roslyn/issues/19523](https://github.com/dotnet/roslyn/issues/19523) to compact more.